tests/controllers/storage.test.ts

```diff
@@ -602,6 +584,954 @@ describe('Storage controller', () => {
 
   });
 
+  describe('Generate share file token', () => {
+
+    it('should fail if `itemId` is not valid', async () => {
+      await testIsErrorWithData('File ID must be a valid string', {
+        params: {},
+        body: {
```

Comment:
Maybe reusing a default body and overriding the desired property the tests could be more explicit + short, p.e: 

`body: { ...defaultBody, views: '1' }`

where `defaultBody` could be:
```
{
  encryptionKey: 'enckey',
  fileToken: 'token',
  bucket: 'bucket'
}
```

As these values seem to be reused in several tests